thrall/app/lib/ElasticSearch.scala

```diff
@@ -168,6 +181,10 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
     """.stripMargin
 
   // Create the exports key or add to it
+  private val replaceUsagesScript =
+    "ctx._source.usages = usages;"
```

Comment:
Not an immediate concern but out of interest, do you have any thoughts on how/if we could use the most recent timestamp in the usages to make sure last-write-wins always even if SQS messages arrive out of order (i.e. discard older messages)? Feels like it would be doable by finding the latest lastModified of both the source and new usages and checking the new one is more recent?

Just thinking ahead, let's not get distracted with that right now.